config/consensus.go

```diff
@@ -108,6 +108,11 @@ type ConsensusParams struct {
 	// each Txn has a MinFee.
 	EnableFeePooling bool
 
+	// EnableAppFeePooling specifies that the sum of fees for application calls
+	// in a group is checked against the sum of the budget for application calls,
+	// rather than check each individual app call is within the budget.
+	EnableAppFeePooling bool
```

Comment:
Let's try to avoid the word Fee in this context.  We'll reserve "fee" for referring to charge imposed by the network, in aglos, for a transaction.  I'd prefer using "Cost" or "Budget" here.  Maybe "EnableAppCostPooling".

Reply:
Agreed - changed to `EnableAppCostPooling `